common/common_functions.sh

```diff
@@ -325,6 +325,13 @@ function load_rdma_modules {
         return $status
     fi
 
+    modprobe ib_umad
```

Comment:
why this is needed?

Reply:
When i did the testing, it turned out that the rdma_cm and rdma_ucm are not enough to load all the rdma resources (namely the issm and the umad resources) necessary for the rping test to pass. and it turned out the ib_umad will load the modules to provide the resources.

This was realized because when the ofed driver container is killed, it unloads all of its modules including the mlx5_core and ib_umad. so a reload is needed to be able to test the rdma after a ofed driver container is killed.